tests/02_eradiate/01_unit/scenes/atmosphere/test_particle_layer.py

```diff
@@ -285,7 +285,36 @@ def test_particle_layer_eval_radprops(modes_all_single, test_dataset):
     )
 
 
-@pytest.mark.parametrize("tau_550", [0.1, 0.5, 1.0, 5.0])
+@pytest.mark.parametrize(
+    "tau_550",
+    np.array([0.6, 1.0, 2.5]) * ureg.dimensionless,
```

Comment:
I would expect conversion to a dimensionless quantity to be seamless, is there a particular reason I missed motivating the complexification of this parametrisation?